src/topojson/topojson_intf.ml

```diff
@@ -264,7 +264,7 @@ module type S = sig
 
   val topojson : t -> topojson
   val bbox : t -> float array option
-
+  val vals_of_t : topojson -> topojson -> float array option -> t
```

Comment:
Ah yes we need to expose constructors for everything in order to build a new value. For the main topojson object we will need:

```ocaml
val v : ?bbox:float array -> topojson -> t
``` 

And then we will need constructors for building `topojson` values too.